packages/atomic/cypress/integration/facets/date-facet/date-facet-assertions.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  convertDateFormatLabel,
+  convertDateToFacetValue,
+  dateField,
+  DateRange,
+  dateRanges,
+} from './date-facet-actions';
+import {convertFacetValueToAPIformat} from '../facet/facet-actions';
+import {dateFacetComponent, FacetSelector} from '../facet/facet-selectors';
+
+export function assertDateFacetValueOnUrl(
+  index = 0,
+  field = dateField,
+  type = dateFacetComponent,
+  valueSeparator = 'to'
+) {
+  it('Should reflected selected facetValue on Url', () => {
+    FacetSelector.facetValueLabelAtIndex(index, field, type)
+      .invoke('text')
+      .then((txt) => {
+        const facetValueInApiFormat = convertFacetValueToAPIformat(
+          txt,
+          type,
+          valueSeparator
+        );
+        const urlHash = `#df[${field.toLowerCase()}]=${encodeURI(
+          facetValueInApiFormat
+        )}`;
+        cy.url().should('include', urlHash);
+      });
+  });
+}
+
+export function assertCustomRangeDisplay(ranges = dateRanges) {
```

Comment:
I would rename here to assertDateRangeDisplay and remove the parameter as its not used